app/models/event_subscriber.rb

```diff
@@ -5,23 +5,13 @@ def subscribed_to?(event)
     enabled
   end
 
-  def self.notify_sync!(event)
-    notify! :sync, event
-  end
-
-  def self.notify_async!(event)
-    notify! :async, event
-  end
-
   def do_request(event)
     Mumukit::Nuntius::Publisher.publish_submissions event
```

Comment:
Something I have not notices previously is that the event subscriber - base class - is publishing always to the submissions queue. This is wrong, and will introduce bugs when new events are added to the queue